enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.ts

```diff
@@ -0,0 +1,37 @@
+import type {
+  MetabaseTheme,
+  EmbeddingThemeOverride,
+  MetabaseColor,
+} from "embedding-sdk/types/theme";
+
+import { colorTuple } from "./color-tuple";
+
+/**
+ * Transforms a public-facing Metabase theme configuration
+ * into a Mantine theme override for internal use.
+ */
+export function getEmbeddingThemeOverride(
+  theme: MetabaseTheme,
+): EmbeddingThemeOverride {
+  const override: EmbeddingThemeOverride = {
+    ...(theme.lineHeight && { lineHeight: theme.lineHeight }),
+    ...(theme.components && { other: theme.components }),
+    ...(theme.fontSize && { fontSizes: { md: theme.fontSize } }),
```

Comment:
@WiNloSt Good point. I was trying to find a way to set the base font size (similar to `body {font-size: ...}`) that would be picked up by other font sizes, but looks like Mantine does not support that.

Should we just expose the `fontSizes` object like in Mantine? The problem is we don't use other text sizes that much in our codebase, so not sure if that will be confusing to SDK users.

Reply:
That depends on how we want to allow customizing other element font sizes. I think having 1 base font size + more knobs for other elements might work because users are not going to understand how modifying `lg` and `sm` would affect their chart.

I'm still not sure how the API should look like yet. From the [Figma file we discussed](https://www.figma.com/file/OamRqg07JOoC4bUqfCkh4t/SDK-Reference-APP?type=design&node-id=2009-4887&mode=design&t=pgcpyogiM44V9x5E-0), maybe we offer
- base `fontSize`
- and a few more options for other texts e.g. values, labels, legends, etc.

That means this change is okay.

Reply:
> I wonder if the base `fontSize` should just apply to the SDK container rather than the `fontSizes.md` property, especially if the fontSizes are already using relative units. Then we don't have to allow changing `fontSizes` at all.

I love the idea. Let's do that!

Reply:
Cool! Will do this in next PR on Tuesday to unblock other PRs.